### PR TITLE
Adstraw/sequence mask

### PR DESCRIPTION
**Pull request by adstraw**
## Description ##
Add support for MXNet SequenceMask op

### Essentials ###
- [x] Passed code style checking (`make lint`)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
Add SequenceMask op

## Comments ##
Passes test_sequence_mask() in test_operator.py

### CODE REVIEW

File: src/ngraph/ngraph_emitter.cc [hunk 1]
```diff
@@ -601,6 +601,33 @@ void Emitter::CreateBinaryOps() {
     return cast_result(CreateAutoBroadcast<ngraph::op::LessEq>(node),
                        getType(node->dtype_));
   };
+  ngraph_op_funcs_["SequenceMask"] = [this](const NodePtr& node) {
+    auto data = op_map_[node->inputs_[0]];
+
+    // if sequence lengths specified
+    if (node->inputs_.size() > 1) {
```
Comment (ashokei) [inline src/ngraph/ngraph_emitter.cc]: can we also check for `use_sequence_length` , though it should be true if size > 1.

File: tests/python/unittest/test_operator.py [hunk 2]
```diff
@@ -4881,5 +4881,4 @@ def test_float16_min_max():
 
 
 if __name__ == '__main__':
-    import nose
-    nose.runmodule()
+    test_sequence_mask()
```
Comment (ashokei) [inline tests/python/unittest/test_operator.py]: may want to revert this.

Comment (adstraw) [reply tests/python/unittest/test_operator.py]: :) I was sure I had.  HAHA.  Will do.

File: src/ngraph/ngraph_emitter.cc (hunk 1 above)
Comment (adstraw) [reply src/ngraph/ngraph_emitter.cc]: ok

**Pull request merged**